### PR TITLE
internal: relocate and fix legacy scripts

### PR DESCRIPTION
**Pull request by ermirizio**
This PR fix classic cloud deployment due to some recent changes and scripts being unmantained for some time.

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/3952/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2022906167)

**Review by alephnull — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/3952/merge
Commit: 6fc4d40205a807cb6288aa473b3ce47166a405e9 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2023294291)

File: bin/dist_build.sh [hunk 1]
```diff
@@ -59,10 +58,8 @@ done
 
 echo "Building Tyk binaries"
 gox -tags 'goplugin' -osarch="linux/amd64 linux/386" -cgo
```
Comment (vesko-tyk) [inline bin/dist_build.sh:60]: If we're removing i386 wouldn't this cause the pipeline to fail when building the binaries?

Comment (ermirizio) [reply bin/dist_build.sh]: removed refs to 386

**Review by asutosh — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: e6dbfa25b8fa503930d577fede5cc0929ce2b224 internal: relocate and fix legacy scripts (#3952)

* internal: relocate and fix legacy scripts
Triggered by: push (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2028173626)

**Pull request merged**